docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -4,200 +4,28 @@ description: Learn in this guide how to deploy your Strapi application on Heroku
-
-### 1. Heroku CLI Installation
-
-Download and install the `Heroku CLI` for your operating system:
-
-:::: tabs card
-
-::: tab Ubuntu
-Run the following from your terminal:
-
-```bash
-sudo snap install --classic heroku
-```
-
-:::
-
-::: tab Mac
-[Download the installer](https://cli-assets.heroku.com/heroku.pkg)
-
-Also available via Homebrew:
-
-```bash
-brew tap heroku/brew && brew install heroku
-```
-
-:::
-
-::: tab Windows
-Download the appropriate installer for your Windows installation:
-
-- [64-bit installer](https://cli-assets.heroku.com/heroku-x64.exe)
-- [32-bit installer](https://cli-assets.heroku.com/heroku-x86.exe)
-  :::
-
-::::
-
-### 2. Login to Heroku from your CLI
-
-Next, you need to login to Heroku from your computer.
-
-```bash
-heroku login
-```
-
-Follow the instructions and return to your command line.
-
-### 3. Create a new project (or use an existing one)
-
-Create a [new Strapi project](/developer-docs/latest/getting-started/quick-start.md) (if you want to deploy an existing project go to step 4).
-
-`Path: ./`
-
-<code-group>
-
-<code-block title="NPM">
-```sh
-npx create-strapi-app@latest my-project --quickstart
-```
-</code-block>
-
-<code-block title="YARN">
-```sh
-yarn create strapi-app my-project --quickstart
-```
-</code-block>
-
-</code-group>
-
-::: tip
-When you use `--quickstart` to create a Strapi project locally, a **SQLite database** is used which is not compatible with Heroku. Therefore, another [database option](#_7-heroku-database-set-up) must be chosen.
-:::
-
-### 4. Update `.gitignore`
-
-Add the following line at end of `.gitignore`:
-
-`Path: ./my-project/.gitignore`
-
-```
-package-lock.json
-```
-
-Even if it is usually recommended to version this file, it may create issues on Heroku.
-
-### 5. Init a Git repository and commit your project
-
-Init the Git repository and commit your project.
-
-`Path: ./my-project/`
-
-```bash
-cd my-project
-git init
-git add .
-git commit -m "Initial Commit"
-```
-
-### 6. Create a Heroku project
-
-Create a new Heroku project.
-
-`Path: ./my-project/`
-
-```bash
-heroku create
-```
-
-You can use `heroku create custom-project-name`, to have Heroku create a `custom-project-name.heroku.com` URL. Otherwise, Heroku will automatically generate a random project name (and URL) for you.
-
-:::tip
-If you have a Heroku project app already created, you would use the following step to initialize your local project folder:
-
-`Path: ./my-project/`
-
-```bash
-heroku git:remote -a your-heroku-app-name
-```
+The purpose of this guide is to allow users to deploy Strapi applications on Heroku. This guide uses the Heroku CLI tool with a PostgreSQL database provided by Heroku. There are other options for how to deploy to Heroku available in the [Heroku documentation](link here).
 
+::: caution
+For security reasons, the Content-type Builder is disabled in production. Changes to the content structure should be developed locally and then deployed to production.
```

Comment:
There are other reasons, can we link to the FAQ entries?

https://docs.strapi.io/developer-docs/latest/getting-started/troubleshooting.html#why-can-t-i-create-or-update-content-types-in-production-staging

https://docs.strapi.io/developer-docs/latest/getting-started/troubleshooting.html#why-are-my-application-s-database-and-uploads-resetting-on-paas